api/api/utils/photon.py

```diff
@@ -120,6 +122,7 @@ def get(
 ) -> HttpResponse:
     logger = parent_logger.getChild("get")
     cache = django_redis.get_redis_connection("default")
```

Comment:
Because this and the `tallies` module are operating on the same conceptual level, should this code also use the tallies connection? This would go hand-in-hand with moving the error handling code into the tallies module itself, though, not part of this change.

I think it makes sense to group these conceptually similar observability utilities.

Reply:
I was thinking about that as well but opted to hold off on that change for now since that's where the previous error counts were going. However, now might be the time to move them to the tallies DB because I've changed the prefixes/key format anyway. Maybe we fast-follow this?